oversteer/gui.py

```diff
@@ -423,10 +423,15 @@ def process_events(self, events):
                         self.test.trigger_action()
                 else:
                     delay = 100
+
+                button = None
+
                 if event.code >= 288 and event.code <= 303:
                     button = event.code - 288
                 if event.code >= 704 and event.code <= 712:
                     button = event.code - 688
+                if event.code >= 304 and event.code <= 316:
+                    button = event.code - 304
```

Comment:
Using this numbering will collide with buttons from 288 to 203. Is that ok? Does the G923 have a different number of buttons than the G29/G920?

Reply:
Unless I am missing something, 288-303 does not collide with 304-316. On that note, I may have missed some buttons as the G923 has the same number of buttons as the G29 (based on what I can tell from screenshots online). The only difference between the G29/G923 and the G920 are the buttons used for the dual clutch system.

Reply:
Buttons with codes 288 and 304 will both be mapped to number 0. Is that correct?

There will be buttons missing but if it's good for you it's good for me. I can't test it.

Reply:
You are correct, this has made it clear to me do I did not understand the above code when I wrote it. Is the code above intended to "normalize" the event code ranges because they would not show up on the UI otherwise? I tested Oversteer without my if statement and all the buttons were recognized as expected. Is that correct or should we map these buttons to 25-37?

Reply:
This function maps event codes 288-303 and 704-712 to buttons with ids 0-24 for the UI. This way they're in a continuous range begining from zero no matter what the wheel codes are and it's easier for the UI to handle.

If the G923 uses the same button codes as the G29 nothing is required. If buttons had different codes something would have to be changed to map the codes correctly to the range 0-24, the order doesn't matter. Only 25 buttons are displayed in the UI, but more can be added manually if there's a need.

Reply:
Makes sense. I think I messed something up with my branch trying to push that change. I will try to fix it.

Reply:
Should be good now.